src/dotnet/commands/dotnet-test/BaseDotnetTestRunner.cs

```diff
@@ -36,11 +36,8 @@ private int DoBuildTestProject(DotnetTestParams dotnetTestParams)
                 $"{dotnetTestParams.ProjectPath}"
             };
 
-            if (dotnetTestParams.Framework != null)
-            {
-                strings.Add("--framework");
-                strings.Add($"{dotnetTestParams.Framework}");
-            }
+            strings.Add("--framework");
```

Comment:
Just to confirm, the framework and runtime are being passed into the projectcontext somewhere else, if they exist, right? So that we still consider the user's input if he gives us one?